src/interface/src/styles/_mixins.scss

```diff
@@ -29,10 +29,26 @@ Declare mixins based on styleguide and apply to classes
   font-weight: 600;
 }
 
+@mixin h3 {
+  @include public-sans();
+  font-size: 22px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 32px;
+}
+
+
 @mixin h4 {
   @include public-sans();
   font-size: 18px;
   font-style: normal;
   font-weight: 600;
   line-height: 32px;
 }
+
+@mixin p {
+  font-size: 16px;
```

Comment:
should we include public-sans here, as well?

Reply:
good catch